app/scripts/modules/core/src/help/help.contents.ts

```diff
@@ -109,6 +109,9 @@ module(HELP_CONTENTS, [])
     'pipeline.config.trigger.webhook.constraints': `
         <p>When provided, only a webhook with a payload containing at least the specified key/value pairs will be allowed to trigger this pipeline. For example, if you wanted to lockdown the systems/users that can trigger this pipeline via this webhook, you could require the key "secret" and value "something-secret" as a constraint.</p>
     `,
+    'pipeline.config.trigger.pubsub.constraints': `
+        <p>When provided, only a pubsub message with a payload containing at least the specified key/value pairs will be allowed to trigger this pipeline. For example, if you wanted to lockdown the systems/users that can trigger this pipeline via this pubsub subscription, you could require the key "secret" and value "something-secret" as a constraint.</p>
```

Comment:
The configured values from the UI will always be strings. I'm assuming you're not doing any type coercion when evaluating equality. What happens if someone's payload key is `true` or a number? They might be a little surprised if this doesn't work.

Just a thought, you don't have to resolve this.

Reply:
Echo converts to a string an does a regex match. So if you have the constraint:

"key": "true",

and the message is

{
"key": true
}

it'll pass

Reply:
Ah cool.

Reply:
word